object_manager.go

```diff
@@ -25,15 +26,15 @@ type IBObjectManager interface {
 type ObjectManager struct {
 	connector IBConnector
 	cmpType   string
-	tenantID  string
+	TenantID  string
```

Comment:
So, if you agree that the NVLocker belongs to Infoblox-go-client (as we discussed in the docker-infoblox review), then we don't need to export tenantID, yes?